transform/mc3/mc3_maf_transform.py

```diff
@@ -61,10 +63,20 @@ def callset_maker(self, allele, source, centerCol, method, line):
         return sample_calls, sample_callsets
 
 
-def transform(mafpath, prefix, workers=5, skip=0, harvest=True, filter=[]):
-    return parent_transform(mafpath, prefix, workers, skip, harvest, filter,
-                            transformer=MC3_MAFTransformer())
+parser = default_argument_parser()
```

Comment:
Should this arg parsing logic be wrapped in an `if main ...` ? 
(Causes tests to fail)

Reply:
Ah, I didn't realize you were importing the transformer code for testing. Makes sense. I will add that back.